### ISSUE TITLE
Easier module deletion for module planner

### ISSUE THREAD

**Issue by linhns** [nusmodifications/nusmods#3281]
### Is your feature request related to a problem? Please describe.

In the module planner, when modules selected occupy the whole screen, it is not easy to drag a module down to delete it

### Describe the solution you'd like

Although there is a dropdown menu containing a remove option, I still feel it is inconvenient.

### Describe alternatives you've considered

Same as above

### Additional context

![image](https://user-images.githubusercontent.com/24732611/119618209-36640200-be35-11eb-943a-a27c81656cb1.png)

As seen in the screenshot, the "drop to remove" space is not seen when more modules are picked

**Comment by chrisgzf** [comment]
Good catch! Thanks for the report @linhns! This is definitely a UX issue we should resolve.

I think what usually happens when it comes to deletion of drag and drop items on apps is:
1. when the item is in a dragged state, a prominent red "deletion zone" that spans across the page occurs either at the top or bottom of the page for people to drag the item to or;
2. you tap on it a few seconds and every deletable item shakes and a (X) icon appears at the corner of every deletable item

Personally I prefer (1). If we are to go with (1), ideally the deletion zone should be at the bottom of the page so that it doesn't clash with the new UI redesign (where the navbar is at the top). @zhuhanming @taneliang any thoughts?
